understanding/21/pointer-gestures.html

```diff
@@ -3,23 +3,39 @@
-			<p>A user may find it difficult or impossible to accomplish these gestures if they have impaired fine motor control, or if they use a specialized or adapted input device such as a head pointer, eye-gaze system, or speech-controlled mouse emulation.</p>
-<p>Note that free-form drag and drop actions are not considered path-based gestures for the purposes of this Success Criterion.</p>
-			<p>Examples of multipoint gestures include a two-finger pinch zoom, a split tap where one finger rests on the screen and a second finger taps, or a two- or three-finger tap or swipe. A user may find it difficult or impossible to accomplish these if they type and point with a single finger or stick, in addition to any of the causes listed above.</p>
-			<p>Authors must ensure that their content can be operated without such complex gestures. When they implement multipoint or path-based gestures, they must ensure that the functionality can also be operated via single-point activation.  Examples of single-point activation on a touchscreen or touchpad include taps, double taps, and long presses. Examples for a mouse, trackpad, head-pointer, or similar device include single clicks, click-and-hold and double clicks.</p>
-			<p>This Success Criterion applies to author-created gestures, as opposed to gestures defined on the level of operating system or user agent. An example for gestures provided on the operating system level would be swiping down to see system notifications, and gestures for built-in assistive technologies (AT) to focus or activate content, or to call up AT menus. Examples of user-agent-implemented gestures would be horizontal swiping implemented by browsers for navigating within the page history, or vertical swiping to scroll page content.</p>
+			<p>The intent of this Success Criterion is to ensure that content can be controlled with a range of pointing devices, abilities, and assistive technologies. Some people  cannot perform gestures in a precise manner, or they may use a specialized or adapted input device such as a head pointer, eye-gaze system, or speech-controlled mouse emulator. Some pointing methods lack the capability or accuracy to perform multipoint or path-based gestures.</p>
+
+			<p>A <strong>path-based gesture</strong> involves an interaction where the user engages a pointer (starting point), carries out a movement that goes through at least one mid-point before disengaging the pointer (end point). The mid-point defines the path as a gesture.</p>
```

Comment:
How about replacing the last sentence with "The mid-point defines the gesture as requiring a specific path, even if the path is not defined for the complete path." This seems backwards as it is.

Reply:
Refining slightly to: The mid-point defines the gesture as requiring a specific path, even if the complete path is not defined.

Reply:
yep, I meant to end with "gesture" but you fixed a different way.